clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/NodeStateChangeChecker.java

```diff
@@ -441,4 +527,14 @@ private Result checkDistributors(Node node, int clusterStateVersion) {
         return allowSettingOfWantedState();
     }
 
+    private Set<Integer> groupsWithStorageNodesWantedStateNotUp() {
+        return clusterInfo.getStorageNodeInfos().stream()
+                          .filter(sni -> !UP.equals(sni.getWantedState().getState()))
```

Comment:
The original method is `otherNodeInGroupHasWantedState`, which differs from this on 2 accounts:
 - Use `getUserWantedState` instead of `getWantedState`?  As it is there is no practical difference, but getUserWantedState corresponds 1:1 with the state we're about approve/deny with the NodeStateChangeChecker.
 - Also check the distributor?  Only matters if an operator is setting (user) wanted state.